chainerrl/links/mlp_bn.py

```diff
@@ -27,15 +29,28 @@ def __call__(self, x):
 
 
 class MLPBN(chainer.Chain):
-    """Multi-Layer Perceptron with BatchNormalization."""
+    """Multi-Layer Perceptron with Batch Normalization.
+
+    Args:
+        in_size (int): Input size.
+        out_size (int): Output size.
+        hidden_sizes (list of ints): Sizes of hidden channels.
+        normalize_input (bool): If set to True, Batch Normalization is applied
+            to inputs.
+        normalize_output (bool): If set to True, Batch Normalization is applied
+            to outputs.
+        nonlinearity (callable): Nonlinearity between layers.
```

Comment:
In most cases, `nonlinearity` is a `chainer.Function`, and it cannot be a `chainer.Link`.  Could you make the followings clear?
- `nonlinearity` has no parameters;
- `nonlinearity`, called with one argument of `chainer.Variable`, returns `chainer.Variable` of the same shape as the input.